src/test/java/com/eleks/academy/whoami/core/state/SuggestingCharacterTest.java

```diff
@@ -28,7 +31,13 @@ void assignCharactersTest() {
 
 		var result = characters.assignCharacters();
 
-		assertNotEquals(inputPlayersAndCharacters, result);
+		List<String> oldValues = new ArrayList<>(inputPlayersAndCharacters.values());
```

Comment:
I would recommend iterating throughout the map, ArrayList doesn't guarantee that element of one list has the same position in another list

Reply:
Resolved